examples/simple/max-restarts.yaml

```diff
@@ -13,14 +13,13 @@ spec:
       spec:
         # Set backoff limit to 0 so job will immediately fail if any pod fails.
         backoffLimit: 0 
-        completionMode: Indexed
         completions: 2
         parallelism: 2
         template:
           spec:
             restartPolicy: Never
```

Comment:
sorry, just saw this, not need for this

Reply:
yep i removed it after i saw the other comment

Reply:
not removed yet :)

Reply:
oh woops sorry I thought this was in reference to the `completionMode: Indexed` - my bad, I removed the restartPolicy so it can be defaulted.